3 - Assignment/240409 - Rust Basic + Roguelike Game, Week 2/geometry/src/lib.rs

```diff
@@ -1,14 +1,23 @@
+
+
 // Calculate the magnitude of a vector by summing the squares of its coordinates
 // and taking the square root. Use the `sqrt()` method to calculate the square
 // root, like `v.sqrt()`.
-fn magnitude(/* ... */) -> f64 {
-    todo!()
+fn magnitude(vec:&[f64;3]) -> f64 {
+    let pow = vec[0].powf(2.0) + vec[1].powf(2.0) + vec[2].powf(2.0);
+    if pow < 0.0 {
+        return -1.0;
+    }
+    return pow.sqrt();
```

Comment:
`return`을 생략할 수 있습니다.

```suggestion
    pow.sqrt()
```